code/modules/projectiles/guns/magic/staff.dm

```diff
@@ -69,6 +71,7 @@
 	item_state = "honker"
 	max_charges = 4
 	recharge_rate = 8
+	harmful = FALSE
```

Comment:
bs

Reply:
Afaik all it does is "slip" you which does no harm (its just a stun)